openvpn-2.4.6-openssl-1.1.0h/client.up.eduvpn.sh

```diff
@@ -1523,23 +1496,20 @@ readonly TB_RESOURCES_PATH="${ARG_TB_PATH}/Contents/Resources"
 #
 #     LEASEWATCHER_PLIST_PATH and REMOVE_LEASEWATCHER_PLIST are passed to the other scripts via the scutil State:/Network/OpenVPN mechanism
 
-# NOT! The functionality from the above comment is disabled
```

Comment:
The changes here from line 1526 to 1543 should be reverted for the lease watch functionality to keep working.